src/shell.js

```diff
@@ -34,13 +34,31 @@ for (var key in Module) {
 
 // The environment setup code below is customized to use Module.
 // *** Environment setup code ***
-var ENVIRONMENT_IS_WEB = typeof window === 'object';
+var ENVIRONMENT_IS_WEB = false;
+var ENVIRONMENT_IS_WORKER = false;
+var ENVIRONMENT_IS_NODE = false;
+
```

Comment:
(shell is the v8 or sm shell, as opposed to node, which is a type of shell, but quite distinct)